integration/js/pages/AppPage.js

```diff
@@ -516,6 +516,12 @@ class AppPage {
     let actualSpeakers = Object.getOwnPropertyNames(actualTranscriptContentBySpeaker);
     let expectedSpeaker = Object.getOwnPropertyNames(expectedTranscriptContentBySpeaker);
 
+    // Temporarily filtering empty speakers for medical transcribe test. Empty speaker issue - P68074811
+    if (isMedicalTranscribe) {
+      console.log(`Filtering empty speaker in medical transcribe. isMedicalTranscribe: ${isMedicalTranscribe}"`);
```

Comment:
Minor: `isMedicalTranscribe` should always be true based on line 520

Reply:
Can you elaborate? 

`isMedicalTranscribe` would determine if the current execution of integration test is for medical or non-medical transcription - https://github.com/aws/amazon-chime-sdk-js/blob/main/integration/js/TranscriptionTest.js#L95-L96. We are applying temporary filter only for medical transcription test.